src/daft-dsl/src/expr.rs

```diff
@@ -3,7 +3,7 @@ use daft_core::{
     count_mode::CountMode,
     datatypes::{try_mean_supertype, try_sum_supertype, DataType, Field, FieldID},
     schema::Schema,
-    utils::supertype::try_get_supertype,
+    utils::{hyperloglog::NUM_REGISTERS, supertype::try_get_supertype},
```

Comment:
you should just export the type instead rather than leaking NUM_REGISTERS

Reply:
Updated!